### ISSUE TITLE
mime-ext.yazi: Unable to preview plain text files without ext

### ISSUE THREAD

**Issue by moxuze** [yazi-rs/plugins#39]
### What system are you running Yazi on?

Linux Wayland

### What terminal are you running Yazi in?

kitty v0.37.0

### `yazi --debug` output

```Shell
Yazi
    Version: 0.4.2 (635bb4be 2024-12-18)
    Debug  : false
    Triple : x86_64-unknown-linux-gnu (linux-x86_64)
    Rustc  : 1.83.0 (90b35a62 2024-11-26)

Ya
    Version: 0.4.2 (635bb4be 2024-12-18)

Emulator
    Brand.from_env      : Some(Kitty)
    Emulator.detect     : Emulator { kind: Left(Kitty), light: false, cell_size: Some((12, 25)) }
    Emulator.detect_full: Ok(Emulator { kind: Left(Kitty), light: false, cell_size: Some((12, 25)) })

Adapter
    Adapter.matches: Kgp

Desktop
    XDG_SESSION_TYPE           : Some("wayland")
    WAYLAND_DISPLAY            : Some("wayland-1")
    DISPLAY                    : Some(":1")
    SWAYSOCK                   : None
    HYPRLAND_INSTANCE_SIGNATURE: Some("5f1df55fcb45a72e92258113685aa9064f6ebf3e_1734532977_202337430")
    WAYFIRE_SOCKET             : None

SSH
    shared.in_ssh_connection: false

WSL
    WSL: false

NVIM
    NVIM          : false
    Neovim version: 0.10.2

Variables
    SHELL           : Some("/usr/bin/zsh")
    EDITOR          : Some("helix")
    VISUAL          : Some("helix")
    YAZI_FILE_ONE   : None
    YAZI_CONFIG_HOME: None
    YAZI_ZOXIDE_OPTS: None
    FZF_DEFAULT_OPTS: None

Text Opener
    default     : Some(Opener { run: "${EDITOR:=helix} \"$@\"", block: true, orphan: false, desc: "edit with $EDITOR or helix", for_: None, spread: true })
    block-create: Some(Opener { run: "${EDITOR:=helix} \"$@\"", block: true, orphan: false, desc: "edit with $EDITOR or helix", for_: None, spread: true })
    block-rename: Some(Opener { run: "${EDITOR:=helix} \"$@\"", block: true, orphan: false, desc: "edit with $EDITOR or helix", for_: None, spread: true })

Multiplexers
    TMUX               : 0
    tmux version       : No such file or directory (os error 2)
    tmux build flags   : enable-sixel=Unknown
    ZELLIJ_SESSION_NAME: None
    Zellij version     : 0.41.2

Dependencies
    file          : 5.46
    ueberzugpp    : No such file or directory (os error 2)
    ffmpeg/ffprobe: 7.1 / 7.1
    pdftoppm      : 24.11.0
    magick        : 7.1.1-41
    fzf           : 0.56.3
    fd/fdfind     : 10.2.0 / No such file or directory (os error 2)
    rg            : 14.1.1
    chafa         : No such file or directory (os error 2)
    zoxide        : 0.9.6
    7z/7zz        : 17.05 / No such file or directory (os error 2)
    jq            : 1.7.1

Clipboard
    wl-copy/paste: 2.2.1 / 2.2.1
    xclip        : No such file or directory (os error 2)
    xsel         : No such file or directory (os error 2)
```

### Describe the bug

Hello, thank you for your awesome plugins! But I've run into a bit of trouble.

TLDR. After installing the plugin `mime-ext.yazi`, I cannot preview a plain text file without an extension, such as `/etc/passwd`:

![241219_03h27m39s](https://github.com/user-attachments/assets/806bda6a-7b31-4a0b-aa77-7d2f7909b91f)

By the way, I'd like to mention that the preview works fine without using this plugin:

![241219_03h38m30s](https://github.com/user-attachments/assets/75bb038f-97bb-4476-a21a-60a1b9ec8b53)

### Minimal reproducer

Step.1 Install `mime-ext.yazi`:

```shell
ya pack -a yazi-rs/plugins:mime-ext
```

Step.2 Add following text to `~/.config/yazi/yazi.toml`:

```toml
[[plugin.prepend_fetchers]]
id   = "mime"
if   = "!(mime|dummy)"
name = "*"
run  = "mime-ext"
prio = "high"
```

Step.3 Open yazi and locate to a plain text file without an extension, such as `/etc/passwd`, then watch the preview.

**Comment by sxyazi** [comment]
No, this is how it works: return mimetype based on the file extension, as in the plugin name.

If you want to get the mimetype with files no ext, use other plugins, or enable `fallback_file1`

Closing as not even a bug

**Comment by moxuze** [comment]
> No, this is how it works: return mimetype based on the file extension, as in the plugin name.
> 
> If you want to get the mimetype with files no ext, use other plugins, or enable `fallback_file1`
> 
> Closing as not even a bug

Well, I set `fallback_file1 = true`, it works fine.

Thank you for your attention.
